plugin-server/src/utils/event.test.ts

```diff
@@ -124,9 +124,8 @@ describe('formPipelineEvent()', () => {
                     }),
                 })
             ),
-        } as any as KafkaMessage
+        } as any as Message
```

Comment:
nit: doesn't look like `formPipelineEvent` is used anywhere, we could remove the function and those tests.